client/matchmaking/find-match.tsx

```diff
@@ -160,6 +155,25 @@ export function FindMatch() {
       contents = assertUnreachable(activeTab)
   }
 
+  let disabledContents: React.ReactNode | undefined
```

Comment:
nit: generally prefer to pull code flows like this into their own component that can do early returns or return a ternary or something